Base/usr/share/man/man2/mount.md

```diff
@@ -37,14 +38,21 @@ mounted file system.
 
 ### Bind mounts
 
-If `MS_BIND` is specified in `flags`, `fs_type` is ignored and a bind mount is performed
-instead. In this case `source` is treated as a path to a file or directory whose contents
-are overlayed over `target`. This can be used  as an alternative to symlinks or hardlinks.
+If `MS_BIND` is specified in `flags`, `fs_type` is ignored and a bind mount is
+performed instead. In this case `source_fd` is treated as a file descriptor to a
+file or directory whose contents are overlayed over `target`. This can be used
```

Comment:
I'd rather say it a bit differently: "in this case, the file or directory specified by `source_fd` is overlayed over `target` — the target appears to be replaced by a copy of the source."